sdk/documentintelligence/Azure.AI.DocumentIntelligence/CHANGELOG.md

```diff
@@ -3,10 +3,46 @@
 ## 1.0.0-beta.4 (Unreleased)
 
 ### Features Added
-
+- Added methods `GetAnalyzeBatchResult`, `GetAnalyzeBatchResults`, `DeleteAnalyzeBatchResult`, and `DeleteAnalyzeResult` to `DocumentIntelligenceClient`.
```

Comment:
This is not the form that we generally recommend for change logs.   We encourage discussing high-level changes/features from the customer interest perspective rather than reporting an API diff.

Reply:
Do you know if this guidance changed recently? Historically, we've used the CHANGELOG in Form Recognizer and Document Intelligence to list changes exhaustively, while we had release notes as blog posts where we highlighted only major changes (here's FR's CHANGELOG for reference: [CHANGELOG](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md)).

I don't feel too strongly about it, but I'm in favor of keeping this pattern for this release since it's what FR customers are used to. We don't have plans to increment this package in the near future, so we could wait until this release is out to follow the appropriate CHANGELOG policy.

Reply:
That's always been the guidance.   We provide no guarantees of API stability for betas.    I've got no objection to keeping the existing pattern here for consistency, but we'll definitely want to adapt for the new packages.